packages/demos/src/custom-logging/index.test.ts

```diff
@@ -18,7 +19,7 @@ describe('integration test: "custom logging"', () => {
   let consoleMessages: ConsoleMessage[];
 
   beforeAll(async () => {
-    server = await startServer(webpackConfigs);
+    server = await startServer(webpackConfigs, nodeWebpackConfig);
```

Comment:
Maybe you could also use an implicit detection via `target: 'node'` instead of this in my opinion unattractive API? This way the user would not have to be burdened with it.